float64 id
float64 dt
nav_msgs/Path path